cfgov/jinja2/v1/sublanding-page/index.html

```diff
@@ -19,6 +19,11 @@
     {% endfor %}
     {% if not hero_exists %}
         {% include 'organisms/sidebar-breakout.html' %}
+        {% for block in page.header %}
+            {% if 'text_introduction' not in block.block_type -%}
```

Comment:
Shouldn't this be `{% if 'featured_content' in block.block_type -%}` so it doesn't render a hero?